app/src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -311,14 +312,21 @@ protected void onCreate(Bundle savedInstanceState) {
         if (getIntent().getBooleanExtra(EXTRA_USE_PROVIDER_AS_WEBLOGIN, false)) {
             webViewLoginMethod = true;
             webloginUrl = getString(R.string.provider_registration_server);
+        }
+        //enabling login url input screen in debug mode
+        //this will be useful in switching the environments during testing
+        else if (BuildConfig.DEBUG) {
+            webViewLoginMethod = false;
         } else {
             webViewLoginMethod = !TextUtils.isEmpty(getResources().getString(R.string.webview_login_url));
             showWebViewLoginUrl = getResources().getBoolean(R.bool.show_server_url_input);
         }
 
         /// load user interface
         if (webViewLoginMethod) {
-            setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+            //setting orientation is not required for NMC
```

Comment:
Actually we should also not do this.
I will create a PR on master: https://github.com/nextcloud/android/pull/11565

Reply:
Removed from this PR.